packages/builder/src/components/backend/DatasourceNavigator/DatasourceNavigator.svelte

```diff
@@ -228,6 +129,13 @@
         {/each}
       {/if}
     {/each}
+    {#if showNoResults}
+      <Layout paddingY="none" paddingX="L">
+        <div class="no-results">
+          There aren't any datasource matching '{searchTerm}'
```

Comment:
Could we update this copy to be more similar to the design section? Maybe just "There aren't any sources matching that name". We could also change the colour to grey to match the design section empty state too.

Reply:
Good catch!